lib/active_encode/engine_adapters/ffmpeg_adapter.rb

```diff
@@ -177,6 +177,22 @@ def cancel(id)
         raise CancelError
       end
 
+      # This method is to clean up files leftover from the ffmpeg encode process.
+      # File names for the pass_through adapter are the same, so this will clean up
+      # pass_through encodes as well.
```

Comment:
This is true and makes me think a little if this is problem with having both the ffmpeg adapter and pass through adapter sharing the same workspace.  You could use one adapter to create an encode and then the other to find it.  But I guess that isn't really a problem since they share the same code for reading the filesystem to fill out the encode object that gets returned.